### PR TITLE
2.51.0 enablement

### PR DESCRIPTION
**Pull request by chiyc**
## Description
This adds the latest and great features from the SDK. The changes depend on the minified mixpanel-js-wrapper being published.

### CODE REVIEW

File: template.tpl [hunk 1]
```diff
@@ -118,6 +122,14 @@ ___TEMPLATE_PARAMETERS___
         "value": "track_links",
         "displayValue": "track_links"
       },
+      {
+        "value": "start_session_recording",
```
Comment (jakewski) [inline template.tpl:126]: can we also add `get_session_recording_properties` while we're here? 🙏

File: template.tpl [hunk 2]
```diff
@@ -1512,7 +1560,7 @@ const libraryName = data.instanceName ? data.instanceName + '.' : '';
 // Build and normalize initialization options map
 const manualOptions = normalizeTable(data.initManualOptions, 'key', 'value') || {};
 
-const initOptions = (data.initOptions === 'manual' ? manualOptions : data.initOptions) || {};
+const initOptions = Object.assign({'mp_loader': 'gtm'}, (data.initOptions === 'manual' ? manualOptions : data.initOptions) || {});
```
Comment (tdumitrescu) [inline template.tpl:1563]: Object.assign is fine with ignoring falsey values, so you could simplify a little bit
```suggestion
const initOptions = Object.assign({'mp_loader': 'gtm'}, data.initOptions === 'manual' ? manualOptions : data.initOptions);
```

File: template.tpl [hunk 3]
```diff
@@ -1739,15 +1787,20 @@ const onsuccess = () => {
         normalizeTable(data.trackFormsLinksProperties, 'name', 'value') || null
       );
       break;
-      
+
+    case 'start_session_recording':
+    case 'stop_session_recording':
```
Comment (tdumitrescu) [inline template.tpl:1792]: it looks like these cases are generally in alphabetical order, so maybe move before `time_event`. alternately, maybe it makes sense to make one single block for all the methods that don't take any arguments?

**Review by tdumitrescu — APPROVED** [review]

File: template.tpl (hunk 3 above)
Comment (chiyc) [reply template.tpl:1792]: I think condensing the zero arity cases makes sense. Will clean this up in addition to the Object.assign recommendation. 👍

**Pull request merged**